src/kolmogorov_solve.jl

```diff
@@ -60,7 +61,14 @@ function DiffEqBase.solve(
         x_sde = hcat(x_sde , u)
     end
     y = phi(x_sde)
+    if use_gpu == true
+        y = y |>gpu
+        xi = xi |> gpu
```

Comment:
Our convention is to match the user intention, so instead adapt it to match `u0`

Reply:
But `u0` will either be a distribution  or there exists no `u0` for KolmogorovPDEProblem?

Reply:
what's the user data that's an array?

Reply:
There's no array input. Two tuples though. `xspan` and `tspan`

Reply:
I thought the only way to know is to take a boolean itself to check if the user wants to use a GPU

Reply:
I see, this is fine then.